addons/sms/models/sms_sms.py

```diff
@@ -96,20 +97,25 @@ def action_set_error(self, failure_type):
     def action_set_outgoing(self):
         self._update_sms_state_and_trackers('outgoing', failure_type=False)
 
-    def send(self, unlink_failed=False, unlink_sent=True, auto_commit=False, raise_exception=False):
+    def send(self, unlink_failed=False, unlink_sent=True, auto_commit=None, raise_exception=False):
         """ Main API method to send SMS.
 
+        This contacts an external server. If the transaction fails, it may be
+        retried which can result in sending multiple SMS messages!
+
           :param unlink_failed: unlink failed SMS after IAP feedback;
           :param unlink_sent: unlink sent SMS after IAP feedback;
-          :param auto_commit: commit after each batch of SMS;
+          :param auto_commit: deprecated, use CRON if needed to commit
```

Comment:
Just remove the parameter. Why allowing people to use a parameter that anyway does not work ? Better force them to update their code directly when migrating to 19.